### PR TITLE
idea for spec review issue template

### PR DESCRIPTION
**Pull request by kwsutter**
Signed-off-by: Kevin Sutter <kwsutter@gmail.com>

This is based off of Scott's original PR:  https://github.com/eclipse/microprofile/pull/193

But, slightly modified to reference a maven staging repository for all of our artifacts.  Needs some more work, but wanted to create this for a discussion with @Emily-Jiang.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/specreview_template.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: \[MP Component] \[Version] Specification Review
+labels: creation,plan,progress,release
+assignees: ''
+---
+
+## Specification issue template
+When creating a specification project lifecycle review, create issues in the [MicroProfile-WG repository](https://github.com/microprofile/microprofile-wg) with the content defined as follows.
+
+- [ ] Specification name and version (two digit):
+- [ ] Add a label from one of: `creation`, `plan`, `progress`, `release`
```
Comment (Emily-Jiang) [inline .github/ISSUE_TEMPLATE/specreview_template.md:13]: I don't think we need all of these labels. As I commented before, I would just add `MPWG-release`.

File: .github/ISSUE_TEMPLATE/specreview_template.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- [ ] The URL of the OSSRH staging repository for the spec (using the staging repository id):
+      (eg, https://oss.sonatype.org/content/repositories/orgeclipsemicroprofile-1357/org/eclipse/microprofile/config/microprofile-config-spec/2.0-RC9/)
+ - [ ] Specification PDF in the form of microprofile-_project_-spec-x.y.pdf where _project_ is the microprofile specification short project name (config, health, ...)
```
Comment (Emily-Jiang) [inline .github/ISSUE_TEMPLATE/specreview_template.md:16]: `microprofile-_project_-spec-x.y-pdf.pdf` as we have to add classifier if we want to push both pdf and html to maven central.

File: .github/ISSUE_TEMPLATE/specreview_template.md (hunk 1 above)
Comment (kwsutter) [reply .github/ISSUE_TEMPLATE/specreview_template.md:13]: I think the labels help identify at a glance what's being requested.  They also help with sorting all of the issues and PRs that will eventually be in this repo.  We'll probably be adding even more labels down the road...

File: .github/ISSUE_TEMPLATE/specreview_template.md (hunk 2 above)
Comment (kwsutter) [reply .github/ISSUE_TEMPLATE/specreview_template.md:16]: Really?  The type of the file is not sufficient to differentiate (.pdf, .html)?  That's kind of ugly...

Comment (kwsutter) [reply .github/ISSUE_TEMPLATE/specreview_template.md:16]: That can't be right...  We already have examples of the same file name being used with different extensions (.jar and .pom):  https://oss.sonatype.org/content/repositories/orgeclipsemicroprofile-1357/org/eclipse/microprofile/config/microprofile-config-tck/2.0-RC9/

Comment (radcortez) [reply .github/ISSUE_TEMPLATE/specreview_template.md:16]: No worries, I was able to remove the classifier name.

**Comment by kwsutter** [comment]
Closing since we decided to focus on the Issue itself and then work backwards to a workable template.  Reference PR #2 with our latest template proposal.

**Pull request closed without merging**